plugins/linux-capture/pipewire.c

```diff
@@ -35,6 +35,7 @@
 #include <spa/debug/format.h>
 #include <spa/debug/types.h>
 #include <spa/param/video/type-info.h>
+#include <spa/utils/result.h>
```

Comment:
Sorry, I forgot to ask you to add a small paragraph to this commit as well citing that this will be used by the next commit to negotiate the stream format properly.